postapi/Controllers/CommentsController.cs

```diff
@@ -8,9 +17,111 @@ namespace postapi.Controllers
     public class CommentsController : ControllerBase
     {
 
+        private readonly IConfiguration _configuration;
+        private readonly IWebHostEnvironment _env;
+        public CommentsController(IConfiguration configuration, IWebHostEnvironment env)
+        {
+            _configuration = configuration;
+            _env = env;
+        }
+
+        [Route("GetComments")]
+        [HttpGet]
+        public JsonResult CommentsGet()
```

Comment:
Can we use try and catch block in every action,
And return type should be IActionResult to handle multiple return type of action method.

Reply:
Updated the code in Comments Branch